supervision/detection/detection_utils.py

```diff
@@ -0,0 +1,82 @@
+from typing import Tuple
+
+import cv2
+import numpy as np
+
+from supervision.detection.core import ORIENTED_BOX_COORDINATES, Detections
+
+
+def scale_detections(
```

Comment:
@onuralpszr can we change function name to specific to letterbox otherwise it will confuse users?

Reply:
@hardikdava  scale_letterbox_detections ? (maybe) what do you think @LinasKo  ?

Reply:
Un-letterboxing feels like a very narrow use case.

We should provide general functions which also enable letterbox reversal, and highlight that case in the examples & docs.

While it might be frustrating, let me take the time to review this in-depth over the weekend & start of next week, for reversing API decisions is hard.

Reply:
@LinasKo  I can add enum parameter and change  other two to src_res and target_res params and based on enum parameter it can be letterbox depended scale or normal scale, I can also add other extra cases to handle normal scale cases without "padding" ?